deps/build.jl

```diff
@@ -1,4 +1,80 @@
 
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
```

Comment:
Could you add this requirement in the [compat] section of Project.toml, please?

Reply:
Done!